test/yamlContext/fixtures/basic.yaml

```diff
@@ -21,18 +21,18 @@ pages:
         slug: 'large-banners'
         as: 'featured-banners'
 
-      take2:
-        type: 'group'
-        slug: 'small-signs'
-        as: 'featured-signs'
+      # take2:
```

Comment:
@tbarho there is a problem with using the adapter name (take2) as key here, it gets overriden by the next time you reuse this adapter :confused:. 
Here YAML treats them as a dictionary.
An alternative spec for this is to replace the key with `as` and add an `adapter` property:
```yaml
pages:
  home:
    needs:
      'featured-banners':
        type: 'group'
        slug: 'large-banners'
        adapter: 'take2'

      'featured-signs':
        type: 'group'
        slug: 'small-signs'
        adapter: 'take2'

      payments:
        call: 'payments.list'
        adapter: 'stripe'

      'pull-requests':
        path: 'http://api.github.com/pulls?user=take2'
        adapter: 'stripe'
```